src/business_logic/state/cached_state.rs

```diff
@@ -14,12 +14,13 @@ use super::{
     state_cache::{StateCache, StorageEntry},
 };
 
+// K: class_hash V: ContractClass
```

Comment:
remove?

Reply:
i wrote it as i kind of documentation, because otherwise it was hard to figure it out what the key should be